tools/mo/unit_tests/mo/frontend_ngraph_test.py

```diff
@@ -76,6 +76,19 @@ def test_main_error_log():
     assert test_log == ref_log
 
 
+def test_rt_info():
+    setup_env()
+    args = [sys.executable, '-m', 'pytest',
+            os.path.join(os.path.dirname(__file__), 'convert/meta_data_test_actual.py'), '-s']
+
+    status = subprocess.run(args, env=os.environ, capture_output=True)
+    print("status.stderr")
+    print(status.stderr.decode("utf-8"))
+    print("status.stdout")
+    print(status.stdout.decode("utf-8"))
```

Comment:
please remove it. Not sure that it is needed.

Reply:
Not needed, removed it.